src/org/zaproxy/zap/extension/bruteforce/BruteForce.java

```diff
@@ -99,6 +99,7 @@ public BruteForce (ScanTarget target, File file, BruteForceListenner listenner,
 	    } else {
 	    	extensions = Collections.emptyList();
 	    }
+	    extsToMissVector = new Vector<>(bruteForceParam.getExtensionsToMissSet());
```

Comment:
I was suggesting change the `extsToMiss` directly, e.g.: `manager.extsToMiss.addAll(bruteForceParam.getExtensionsToMissSet())` (which avoids changing `setupManager`).

Reply:
Wow, it is beautiful.